README.rst

```diff
@@ -479,8 +446,129 @@ You may want to set these up via cron to run regularly::
     * * * * * (cd $PROJECT; python manage.py send_queued_mail --processes=1 >> $PROJECT/cron_mail.log 2>&1)
     0 1 * * * (cd $PROJECT; python manage.py cleanup_mail --days=30 --delete-attachments >> $PROJECT/cron_mail_cleanup.log 2>&1)
 
+
+Integration with Celery
+=======================
+
+If your Django project runs in a Celery enabled configuration, you can use its worker to send out
+queued emails. Compared to the solution with cron (see above), or the solution with uWSGI timers
+(see below) this setup has the big advantage that queued emails are send *immediatly* after they
+have been added to the mail queue. The delivery is still performed in a separate and asynchronous
+task, which prevents sending emails during the request/response-cycle.
+
+If you `configured Celery`_ in your project and started the `Celery worker`_,  you should see
+something such as:
+
+.. code-block:: text
+
+	--------------- celery@halcyon.local v4.0 (latentcall)
+	--- ***** -----
+	-- ******* ---- [Configuration]
+	- *** --- * --- . broker:      amqp://guest@localhost:5672//
+	- ** ---------- . app:         __main__:0x1012d8590
+	- ** ---------- . concurrency: 8 (processes)
+	- ** ---------- . events:      OFF (enable -E to monitor this worker)
+	- ** ----------
+	- *** --- * --- [Queues]
+	-- ******* ---- . celery:      exchange:celery(direct) binding:celery
+	--- ***** -----
+
+	[tasks]
+	. post_office.tasks.cleanup_expired_mails
+	. post_office.tasks.send_queued_mail
+
+
+Emails will now be delivered by the Celery worker, immediatly after they have been queued. In order
+to make this happen, the project's ``celery.py`` setup shall invoke the autodiscover_tasks_
+function. There is no need to otherwise configure Post Office for integrating with Celery. However,
+in case of a temporary delivery failure, we might want retrying to send those emails by a periodic
+task. This can be done by a simple `Celery beat configuration`_, for instance through
+
+.. code-block:: python
+
+	app.conf.beat_schedule = {
+	    'send-queued-mail': {
+	        'task': 'post_office.tasks.send_queued_mail',
+	        'schedule': 600.0,
+	    },
+	}
+
+This will (re-)send queued emails every 10 minutes, which is handy in case the first (and immediate)
```

Comment:
I think we should reword this to `This will send queued emails every 10 minutes.` and not mention resending when delivery has failed.